examples/lambda-secretLayer/README.md

```diff
@@ -0,0 +1,53 @@
+# secret-lambdaLayer:
+
+This application will create an SSM layer to use in our integrations.
```

Comment:
You will need to deploy one layer per AWS Region you want to use
Currently the layer support only NodeJS runtimes.